### PR TITLE
Update variable to background file

### PR DESCRIPTION
**Pull request by Shehu-Yakubu**

### CODE REVIEW

**Comment by Shehu-Yakubu** [comment]
The file has been updated by another developer, but a simple reassignment to the variable type is made from (var to const). Would that make any difference?

File: background.js [hunk 1]
```diff
@@ -9,15 +9,19 @@ browser.runtime.onMessage.addListener(
 });
 
 // uninstall feedback redirect
-var isFirefox = typeof InstallTrigger !== 'undefined';
+const isFirefox = typeof InstallTrigger !== 'undefined';
+
+// The variable (isEdge) has been updated successfully by the (isFirefox) variable above
```
Comment (loveshdongre) [inline background.js:14]: I'm not sure why this is mentioned

File: background.js [hunk 2]
```diff
@@ -9,15 +9,19 @@ browser.runtime.onMessage.addListener(
...
 // var isEdge = !isIE && !!window.StyleMedia;
 
 if(isFirefox) {
     browser.runtime.setUninstallURL("https://addons.mozilla.org/en-US/firefox/addon/leetcode-enhancer/"); // mozilla feedback url
 }
+
```
Comment (loveshdongre) [inline background.js:20]: Why don't need to add extra line

**Comment by loveshdongre** [comment]
Closing this since no activity

**Pull request closed without merging**